Unosquare.CodeChallenge/WarehouseRESTfulAPI/Controllers/companies.cs

```diff
@@ -22,7 +22,7 @@ public companies(IWarehouseService<CompaniesServices> companyservice)
         [HttpGet]
         public IActionResult getAllCompanies()
         {
-            return  this.Ok(JsonSerializer.Serialize(companyService.getAllCompanies()));
```

Comment:
.NET 6 already serializes the output,

this.Ok is redundant, change to Ok only

Reply:
ok .. I'll make the changes.